tests/regression/regression.py

```diff
@@ -24,16 +25,64 @@
 #
 
 
-from pyBeam import CBeamSolver
+import pdb
 import numpy as np
+import sys, os
+sys.path.append(str(os.path.realpath(__file__))[:-31] + '/pyBeam')
+sys.path.append(str(os.path.realpath(__file__))[:-31] + '/pyBeam/python')
+#sys.path.append('../../pyBeam')
+#sys.path.append('../../pyBeam/python')
+import in_out
+import swig
 
-beam = CBeamSolver()
+def Input_parsing(BEAM_config, inputs):  
```

Comment:
This should become a generic function in the pyBeam library

Reply:
I'll take care of that